src/components/Editor/DebugLine.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import { Component } from "react";
+import type { SourceEditor } from "devtools-source-editor";
+import type { Frame } from "../../types";
+
+type DebugLineProps = {
+  editor: SourceEditor,
+  frame: Frame,
+  visibleSourceId: string
```

Comment:
I see, perhaps we want to encapsulate some of this knowledge into the component so it knows what to do... 

What if we passed the selected source in and DL checked to see if it was loading? If it's loading we don't highlight...